packages/web/src/services/AudiusBackend.ts

```diff
@@ -1911,24 +1925,24 @@ class AudiusBackend {
     return audiusLibs.Account.changePassword(email, password, oldpassword)
   }
 
-  static async confirmCredentials(email: string, password: string) {
+  async function confirmCredentials(email: string, password: string) {
     await waitForLibsInit()
     return audiusLibs.Account.confirmCredentials(email, password)
   }
 
-  static async sendRecoveryEmail() {
+  async function sendRecoveryEmail() {
     await waitForLibsInit()
-    const host = AudiusBackend._getHostUrl()
+    const host = _getHostUrl()
     return audiusLibs.Account.generateRecoveryLink({ host })
   }
 
-  static _getHostUrl() {
-    return NATIVE_MOBILE && process.env.REACT_APP_ENVIRONMENT === 'production'
-      ? AUDIUS_ORIGIN
+  async function _getHostUrl() {
+    return nativeMobile && process.env.REACT_APP_ENVIRONMENT === 'production'
```

Comment:
i kinda forget if process exits like this in native mobile, seemingly REACT_APP wouldn't since thats a CRA thing?

Reply:
Yeah I just missed this one, updated now!